src/commands/asset.rs

```diff
@@ -23,7 +23,7 @@ impl AssetCommand {
     pub fn list(&self) -> anyhow::Result<Assets, CommandError> {
         Ok(Assets::new(commands::get(
             &self.authentication,
-            "v4/assets",
+            "v4/assets?type=in.('appweb','audio','edge-app','image','video','web')",
```

Comment:
Why not say everything but edge app files ? If we add new asset type it won't be shown by default

Reply:
> If we add new asset type it won't be shown by default

Yes, we want exactly this approach.

Reply:
We should remove `appweb`. But really, we should filter out edge-app files in the API rather than as a filter.